dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -24778,1696 +24779,2928 @@ BEGIN
+    64178 "Ideograful de compatibilitate CJK fab2"
+    64179 "Ideograful de compatibilitate CJK fab3"
+    64180 "Ideograful de compatibilitate CJK fab4"
+    64181 "Ideograful de compatibilitate CJK fab5"
+    64182 "Ideograful de compatibilitate CJK fab6"
+    64183 "Ideograful de compatibilitate CJK fab7"
+    64184 "Ideograful de compatibilitate CJK fab8"
+    64185 "Ideograful de compatibilitate CJK fab9"
+    64186 "Ideograful de compatibilitate CJK faba"
+    64187 "Ideograful de compatibilitate CJK fabb"
+    64188 "Ideograful de compatibilitate CJK fabc"
+    64189 "Ideograful de compatibilitate CJK fabd"
+    64190 "Ideograful de compatibilitate CJK fabe"
+    64191 "Ideograful de compatibilitate CJK fabf"
+    64192 "Ideograful de compatibilitate CJK fac0"
+    64193 "Ideograful de compatibilitate CJK fac1"
+    64194 "Ideograful de compatibilitate CJK fac2"
+    64195 "Ideograful de compatibilitate CJK fac3"
+    64196 "Ideograful de compatibilitate CJK fac4"
+    64197 "Ideograful de compatibilitate CJK fac5"
+    64198 "Ideograful de compatibilitate CJK fac6"
+    64199 "Ideograful de compatibilitate CJK fac7"
+    64200 "Ideograful de compatibilitate CJK fac8"
+    64201 "Ideograful de compatibilitate CJK fac9"
+    64202 "Ideograful de compatibilitate CJK faca"
+    64203 "Ideograful de compatibilitate CJK facb"
+    64204 "Ideograful de compatibilitate CJK facc"
+    64205 "Ideograful de compatibilitate CJK facd"
+    64206 "Ideograful de compatibilitate CJK face"
+    64207 "Ideograful de compatibilitate CJK facf"
+    64208 "Ideograful de compatibilitate CJK fad0"
+    64209 "Ideograful de compatibilitate CJK fad1"
+    64210 "Ideograful de compatibilitate CJK fad2"
+    64211 "Ideograful de compatibilitate CJK fad3"
+    64212 "Ideograful de compatibilitate CJK fad4"
+    64213 "Ideograful de compatibilitate CJK fad5"
+    64214 "Ideograful de compatibilitate CJK fad6"
+    64215 "Ideograful de compatibilitate CJK fad7"
+    64216 "Ideograful de compatibilitate CJK fad8"
+    64217 "Ideograful de compatibilitate CJK fad9"
+    64256 "Ligatura mică latină ff"
+    64257 "Ligatura mică latină fi"
+    64258 "Ligatura mică latină fl"
+    64259 "Ligatura mică latină ffi"
+    64260 "Ligatura mică latină ffl"
+    64261 "Ligatura mică latină lungă s t"
+    64262 "Ligatura mică latină st"
+    64275 "Ligatura mică armenească men now"
+    64276 "Ligatura mică armenească men ech"
+    64277 "Ligatura mică armenească men ini"
+    64278 "Ligatura mică armenească vew now"
+    64279 "Ligatura mică armenească men xeh"
+    64285 "Litera evreiască yod cu hiriq"
+    64286 "Punctul evreiesc iudeo-spaniol varika"
+    64287 "Litera evreiască yod yod patahul idiș"
+    64288 "Litera evreiască ayinul alternativ"
+    64289 "Litera evreiască aleful mare"
+    64290 "Litera evreiască daletul mare"
+    64291 "Litera evreiască heul mare"
+    64292 "Litera evreiască kaful mare"
+    64293 "Litera evreiască şchiopul cel mare"
```

Comment:
I think "Lamed" in English refers to its literal proper name, not to an adjective. See the Wikipedia page: https://en.wikipedia.org/wiki/Lamedh